common/source/docs/common-lua-scripts.rst

```diff
@@ -289,6 +366,35 @@ The relay library proivdes access to controlling relay outputs.
 
 - :code:`toggle(relay_num)` - Toggles the requested relay on or off.
 
+
+Servo_Channels (SRV_Channels:)
```

Comment:
No underscore, this is the human friendly output.

Reply:
fixed